README.md

```diff
@@ -66,11 +88,11 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
-[] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.querySelectorAll(".post")[0].innerHTML`
-[] `document.querySelectorAll(".post").innerHTML`
+[x] `$(".post").css("background", "peachpuff")`
+[] `$(".post").innerHTML`  
+[x] `$(".post").html()`
+[x] `document.querySelectorAll(".post")[0].innerHTML`
+[x] `document.querySelectorAll(".post").innerHTML`
 ```
```

Comment:
We can't run that last one. `querySelectorAll(".post")` returns a collection of DOM elements. `.innerHTML` can only be called on a single Vanilla JS DOM element.